gitpod-io/website
# :bug: fix: UI Bug for pricing calculator slider

## Description

This Fixes a :bug: (for specific users) in Pricing Calculator Slider


|Before|After|
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/55068936/214805824-c771cb53-3e7e-46cf-915e-c6733feba7c0.png)|![image](https://user-images.githubusercontent.com/55068936/214805921-0ede4e4d-8db9-44b9-b952-117d74d85ffc.png)|

## Related Issue(s)



Fixes [Internal Conversastion](https://gitpod.slack.com/archives/C01KGM9ETU6/p1674722736954699?thread_ts=1674471516.434459&cid=C01KGM9ETU6)

## How to test



- Open `/pricing` & check slider :smile: 


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3308"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
